### DISCUSSION TITLE
MIP-NERF taking me to Weights & Biases

### DISCUSSION THREAD

**Discussion by iruiz85** [nerfstudio-project/nerfstudio#2170] [General]
Hi there, 

I'm pretty new on NERFStudio and checking the different models provided by NERFStudio I've checked MIP-NERF that's taking me to wandb.ai portal, but I have no clue about how the 3D model looks like, no idea where can I find it.

Any idea?
Thanks in advance!

**Comment by tancik** [comment]
Mipnerf is too slow to run in the viewer. You can only view rendered images or video.
